test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -58,6 +58,8 @@ def test_ec2_tensorflow_inference_eia_cpu(tensorflow_inference_eia, ec2_connecti
 @pytest.mark.parametrize("ec2_instance_type", TF_EC2_GPU_INSTANCE_TYPE, indirect=True)
 @pytest.mark.parametrize("ei_accelerator_type", TF_EC2_EIA_ACCELERATOR_TYPE, indirect=True)
 def test_ec2_tensorflow_inference_eia_gpu(tensorflow_inference_eia, ec2_connection, region, eia_only):
+    if ec2_instance_type in ["p4d.24xlarge"]:
```

Comment:
`ec2_instance_type` is a parametrized variable with a single value, and is not a list.

It will be parametrized with each individual value in the list `TF_EC2_GPU_INSTANCE_TYPE`.

Therefore, what you will need to do is this:
```suggestion
def test_ec2_tensorflow_inference_eia_gpu(tensorflow_inference_eia, ec2_connection, ec2_instance_type, region, eia_only):
    if ec2_instance_type == "p4d.24xlarge":
```